AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5541,6 +5540,7 @@ rejestracja.pwik.gliwice.pl#%#AG_onLoad(function(){var a=document.querySelector(
 login.e-dowod.gov.pl#%#document.cookie="acceptRodoSie=hide; path=/;";
 patienten.kvno.de#%#(function() { try { if ("undefined" != typeof localStorage) { localStorage.setItem("cookies-confirmed", "true") } } catch (ex) {} })();
 creditkarma.co.uk#%#document.cookie = "cc_cookie_accept=cc_cookie_accept";
+wetter.com#%#AG_onLoad(function() {setTimeout(function() {var a = document.querySelector('a.cmp-lnk-accept-close');a&&a.click();},1000)})
```

Comment:
This rule doesn't work for me because timeout is too little, with 2000 it works ok.
But I think it's better to use mutationObserver if it possible

Reply:
@stanislav-atr

Reply:
@Sergey-Lyapin, i've increased interval to 2000. Using mutationObserver blocks the page making elements unclickable for some reason.